main.go

```diff
@@ -8,6 +8,9 @@ import (
 	"strconv"
 	"time"
 
+	// pprof can be removed later
+	_ "net/http/pprof"
```

Comment:
Good to remove now :slightly_smiling_face:

Reply:
Removed & re-pushed